Apps/Inc/ReadCarCAN.h

```diff
@@ -9,4 +9,12 @@
 
 #include "CANbus.h"
 
+#define SAT_BUF_LENGTH 5
+
+/**
+ * @brief Returns whether regen braking is on or off
+ * @return  Whether regen braking is on or off
```

Comment:
It's more whether regen is allowed since this thread doesn't explicitly control the actuation of the system.